source/frontend/src/features/mapSideBar/property/tabs/propertyDetails/detail/propertyOperation/columns.tsx

```diff
@@ -9,80 +9,87 @@ import { convertArea, formatNumber } from '@/utils';
 
 import { PropertyOperationResult } from './OperationView';
 
-const columns: ColumnWithProps<PropertyOperationResult>[] = [
-  {
-    Header: 'Parent',
-    accessor: 'isSource',
-    align: 'center',
-    width: 5,
-    maxWidth: 5,
-    Cell: (props: CellProps<PropertyOperationResult>) => {
-      return props.row.original.isSource ? (
-        <span data-testid="isSource">
-          <FiCheck size="2rem" color="black" />
-        </span>
-      ) : (
-        <></>
-      );
+const getColumns: (isSubdivision: boolean) => ColumnWithProps<PropertyOperationResult>[] = (
+  isSubdivision: boolean,
+) => {
+  return [
+    {
+      Header: isSubdivision ? 'Parent' : 'Child',
+      accessor: 'isSource',
+      align: 'center',
+      width: 5,
+      maxWidth: 5,
+      Cell: (props: CellProps<PropertyOperationResult>) => {
+        const isSource = props.row.original.isSource;
+        if ((isSubdivision && isSource) || (!isSubdivision && !isSource)) {
```

Comment:
isSubdivision === isSource 
?

Reply:
Yeah, logic wise I agree is the same. But I think is more readable by separating both sections.